### DISCUSSION TITLE
Can ModernFlyouts (v.0.11) override the previous installation?

### DISCUSSION THREAD

**Discussion by tylvn** [ShankarBUS/ModernFlyouts#1091] [Q&A]
I wonder if the new version of ModernFlyouts (v.0.11) can override the installation of v0.10.0-beta.1 and v0.9.3 stable versions, both of which exist currently on my computer at the same time making it impossible for me to distinguish between them.

<img width="573" alt="image" src="https://user-images.githubusercontent.com/95011132/190157568-018be970-e024-45c1-92d5-e5abb14e65f1.png">

**Comment by HeyItsJono** [comment]
Where did you get 0.11?

**Reply by tylvn** [reply]
I don't have v0.11, I just only have the installation of v0.10.0-beta.1 and v0.9.3 stable versions.

**Comment by Samuel12321** [comment]
0.11 hasn't been released yet.
It should remove previous versions.

**Reply by tylvn** [reply]
Thanks for your answer.
